2022q4/freebsd-foundation.adoc

```diff
@@ -0,0 +1,184 @@
+
+* Sponsored and helped organize the November 2022 FreeBSD Vendor Summit. Videos
+from the Summit are available link:https://youtube.com/playlist?list=PLugwS7L7NMXwVfBq5eDRky450jp7LTRJj[here].
+
+* Held a new FreeBSD Friday: link:https://youtu.be/t2VLTtHYIcA[An Introduction to FreeBSD Services] by Drew
+  Gurkowski
+
+* Published the
+  link:https://freebsdfoundation.org/news-and-events/newsletter/freebsd-foundation-fall-2022-update/[Fall]
+  and
+  link:https://freebsdfoundation.org/news-and-events/newsletter/11982/[Winter]
+  Newsletter updates
+
+* New Blog Posts
+
+** link:https://freebsdfoundation.org/blog/meet-the-2022-freebsd-google-summer-of-code-students-koichi-imai/[Meet the 2022 FreeBSD Google Summer of Code Students: Koichi Imai]
+** link:https://freebsdfoundation.org/blog/meet-the-2022-freebsd-google-summer-of-code-students-bojan-novkovic/[Meet the 2022 FreeBSD Google Summer of Code Students: Bojan Novković]
+** link:https://freebsdfoundation.org/blog/keeping-freebsd-secure-learn-the-whys-and-hows-with-the-freebsd-sec-team/[Keeping FreeBSD Secure: Learn the Whys and Hows with the FreeBSD Sec Team]
+** link:https://freebsdfoundation.org/blog/the-freebsd-journal-is-still-free/[The FreeBSD Journal is still Free!]
+** link:https://freebsdfoundation.org/blog/eurobsdcon-2022-trip-report-muhammad-moinur-rahman/[EuroBSDCon 2022 Trip Report: Muhammad Moinur Rahman]
+** link:https://freebsdfoundation.org/blog/eurobsdcon-2022-trip-report-patrick-mcevoy/[EuroBSDCon 2022 Trip Report: Patrick McEvoy]
+** link:https://freebsdfoundation.org/blog/fall-foundation-software-development-update/[Fall Foundation Software Development Update]
+** link:https://freebsdfoundation.org/blog/invest-in-freebsd/[Invest in FreeBSD]
+** link:https://freebsdfoundation.org/blog/2022-in-review-advocacy/[2022 in Review: Advocacy]
+** link:https://freebsdfoundation.org/blog/foundation-sponsors-update-to-wireguard-kernel-port-for-freebsd/[Foundation Sponsors Update to WireGuard Kernel Port for FreeBSD]
+** link:https://freebsdfoundation.org/blog/2022-in-review-fundraising-update/[2022 in Review: Fundraising Update]
+** link:https://freebsdfoundation.org/blog/2022-in-review-software-development/[2022 in Review: Software Development]
+** link:https://freebsdfoundation.org/blog/2022-in-review-continuous-integration-and-quality-assurance-update/[2022 in Review: Continuous Integration and Quality Assurance Update]
+
+* FreeBSD in the news:
+
+** link:https://freebsdfoundation.org/news-and-events/latest-news/ampere-getting-cloud-native-with-freebsd-on-oci-ampere-a1-with-terraform/[Ampere: Getting Cloud-Native with FreeBSD on OCI Ampere A1 with Terraform]
+** link:https://freebsdfoundation.org/news-and-events/latest-news/freebsd-is-well-supported-on-4th-gen-amd-epyc-processors/[FreeBSD is Well Supported on 4th Gen AMD EPYC™ Processors]
+
+* For a quick review of all the Foundation efforts in 2023, check out our link:https://youtu.be/6ybbeFXFm-I[2023 Thank You Video].
```

Comment:
Do you mean 2022 instead of 2023 here? (2 occurences.)

Reply:
Resolved.  Thanks.